rclcpp/include/rclcpp/executors/multi_threaded_executor.hpp

```diff
@@ -47,7 +47,7 @@ class MultiThreadedExecutor : public rclcpp::Executor
    *
    * \param options common options for all executors
    * \param number_of_threads number of threads to have in the thread pool,
-   *   the default 0 will use the number of cpu cores found instead
+   *   the default 0 will use the number of cpu cores found at least 2+
```

Comment:
```suggestion
   *   the default 0 will use the number of cpu cores found (minimum of 2)
```

Reply:
@clalancette 👍 thanks for checking, i will change it as suggested.